app/views/users/_profile.html.erb

```diff
@@ -164,13 +164,13 @@
       </label>
       <p class="crayons-field__description"><%= t("views.logo.color_1.description") %></p>
       <div class="flex items-center w-100 m:w-50">
-        <%= f.public_send("text_field", "users_setting[brand_color1]",
```

Comment:
All of the changes in this file must have come from a Rubocop run in the pre-commit hook 🤷‍♂️